packages/task/src/browser/task-service.ts

```diff
@@ -947,7 +976,9 @@ export class TaskService implements TaskConfigurationClient {
             }
         );
         this.shell.addWidget(widget, { area: 'bottom' });
-        this.shell.activateWidget(widget.id);
+        if (taskInfo && taskInfo.config.presentation && taskInfo.config.presentation.reveal === RevealKind.Always) {
+            this.shell.activateWidget(widget.id);
```

Comment:
Other cases looks good to me, except this one. Are you sure that we should always focus?

btw inlining `showTaskTerminal` was a good decision, code looks easy to understand

Reply:
You are right. I tested in vs code. In this case we should also check the value of `focus`